core/common/src/main/java/org/phenotips/remote/common/internal/RemoteMatchedPatientClusterView.java

```diff
@@ -148,6 +149,18 @@ private boolean requestedEmpty(final int fromIndex, final int toIndex)
      */
     private JSONArray buildMatchesJSONArray(final int fromIndex, final int toIndex)
     {
+        // Sort by score
+        Collections.sort(this.remoteMatches, new Comparator<RemotePatientSimilarityView>()
+        {
+            @Override
+            public int compare(RemotePatientSimilarityView o1, RemotePatientSimilarityView o2)
+            {
+                double score1 = (Double) o1.getScore();
+                double score2 = (Double) o2.getScore();
```

Comment:
Why the cast to/from `Double`? It looks to me like `getScore` returns a `double`, yes?